Katello/katello
# Fixes #34719 - Remove legacy Ansible Collections code.

#### What are the changes introduced in this pull request?

* The new UI is using Ansible Collections for the backend, this PR addresses old legacy code cleanup.

#### Considerations taken when implementing this change?

* Not breaking the New UI

#### What are the testing steps for this pull request?

* Check out the PR
* Navigate around doing Ansible Collections stuff and verify there are no console errors
